lighthouse-core/report/v2/report-generator.js

```diff
@@ -97,11 +98,14 @@ class ReportGeneratorV2 {
   generateReportHtml(reportAsJson) {
     const sanitizedJson = JSON.stringify(reportAsJson).replace(/</g, '\\u003c');
     const sanitizedJavascript = REPORT_JAVASCRIPT.replace(/<\//g, '\\u003c/');
+    // Remove script in templates files just to be safe.
+    const sanitizedTemplates = REPORT_TEMPLATES.replace(/<script>([\s\S]*?)<\/script>/g, '');
```

Comment:
I agree with you. I recall that `el.innerHTML = '<div>just a string</div>';` was deemed unsafe and that's our own markup :) The argument from _someonnnnne_ was that we may add things later and not realize the mistakes.